stopHadoop.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+/opt/hadoop-3.4.0/sbin/stop-yarn.sh
```

Comment:
These paths are only going to work for hadoop 3.4 installed in that dir. Use HADOOP_HOME, or make an env file